modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/ActivityQuestionRow.es.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import classNames from 'classnames';
+import React, {useMemo} from 'react';
+import {Link} from 'react-router-dom';
+
+import {
+	ActivityBody,
+	ActivityFooter,
+	ActivityHeader,
+	ActivityHeaderBadge,
+	MESSAGE_TYPES,
+} from './ActivityUI.es';
+
+export default function ActivityQuestionRow({
+	context,
+	creatorId,
+	currentSection,
+	linkProps,
+	question,
+	rowSelected,
+}) {
+	const {headline} = question;
+
+	const sectionTitle =
+		currentSection || currentSection === '0'
+			? currentSection
+			: question.messageBoardSection &&
+			  question.messageBoardSection.title;
+
+	const creatorInformation = question.creator
+		? {
+				link: `/questions/all/creator/${question.creator.id}`,
+				name: question.creator.name,
+				portraitURL: question.creator.image,
+				userId: String(question.creator.id),
+		  }
+		: {
+				link: `/questions/${sectionTitle}`,
+				name: '',
+				portraitURL: '',
+				userId: '0',
+		  };
+
+	const isRowSelected = question.friendlyUrlPath === rowSelected;
+
+	const badgeField = useMemo(() => {
```

Comment:
Hi @juliocezar96 , can you replace all `badgeField` for `messageType` ? I think this name has a best fit than the other.

```suggestion
	const messageType = useMemo(() => {
```